great-idea/css/index.css

```diff
@@ -3,64 +3,305 @@
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
 }
 /* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
-	display: block;
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
 }
 body {
-	line-height: 1;
+  line-height: 1;
 }
-ol, ul {
-	list-style: none;
+ol,
+ul {
+  list-style: none;
 }
-blockquote, q {
-	quotes: none;
+blockquote,
+q {
+  quotes: none;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
-	content: '';
-	content: none;
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
 }
 table {
-	border-collapse: collapse;
-	border-spacing: 0;
+  border-collapse: collapse;
+  border-spacing: 0;
 }
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+  box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+html,
+body {
+  height: 100%;
+  font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Bangers", cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+  margin: 0 auto;
+  width: 52%;
+}
+header {
+  display: flex;
+  width: 100%;
+  height: 40px;
+  margin-bottom: 50px;
+}
+header nav {
+  flex-direction: row;
+  top: 50%;
+  font-size: 16px;
+  line-height: 1.5;
+}
+header nav a {
+  padding: 1px 5px;
+  margin: 5px 50px 0 0;
+  text-decoration: none;
+  color: gray;
+}
+header img {
+  justify-content: flex-end;
+}
+article {
+  display: flex;
+  margin-top: 100px;
+  width: 100%;
+  height: 400px;
+  margin-bottom: 30px;
+}
+article div {
+  display: flex;
+  width: 100%;
+  justify-content: flex-start;
+}
+article h1 {
+  flex-direction: row;
+  height: 220px;
+  width: 310px;
+  align-content: center;
+  font-size: 70px;
+  text-align: center;
+  padding: 10px 20px;
+}
+div h2 {
+  border: 1px solid black;
+  font-family: Georgia, "Times New Roman", Times, serif;
+  height: 40px;
+  width: 200px;
+  text-align: center;
+  margin-left: 20%;
+  display: inline-block;
+  position: absolute;
+  top: 45%;
+  left: 7%;
+  padding-top: 10px;
+}
+
+div img {
+  display: flex;
+  flex-direction: flex-end;
+}
+section.top-section {
+  width: 100%;
+  display: flex;
+  margin-top: 50px;
+  margin-bottom: 50px;
 }
 
```

Comment:
Your text columns could use a little space between. My recommendation is to use percentages and leave a little 'gutter space' left over and then do justify content space between. For example: if you set your features div and your about div to 49% and added justify content space between to the top section, that would naturally create a margin.